remotes/fixup.go

```diff
@@ -79,32 +81,35 @@ func fixupImage(ctx context.Context, baseImage bundle.BaseImage, ref reference.N
 	return baseImage, nil
 }
 
-func fixupBaseImage(ctx context.Context, baseImage *bundle.BaseImage, ref reference.Named, resolver docker.ResolverBlobMounter) (repoOnly reference.Named, imageRef reference.Named, descriptor ocischemav1.Descriptor, err error) {
+func fixupBaseImage(ctx context.Context,
+	baseImage *bundle.BaseImage,
+	ref opts.NamedOption,
+	resolver docker.ResolverBlobMounter) (repoOnly reference.Named, imageRef reference.Named, descriptor ocischemav1.Descriptor, err error) {
 	var e error
 	if e = checkBaseImage(baseImage); e != nil {
 		err = fmt.Errorf("invalid image %q: %s", ref, e)
-		return
+		return nil, nil, descriptor, err
```

Comment:
There are named returns, so returning without any value is ok. Maybe the linter complains? But then it shouldn't, or it should complain about named returns instead 😄

Reply:
If I remember well, the linter complains about the named rets itself.
And I think that like this the code is in the same style as the rest of the project.

Reply:
Well maybe then remove also the named returns?

Reply:
Done!